book/src/volume-cloning.md

```diff
@@ -0,0 +1,45 @@
+# Volume Cloning
+
+## Status
+
+* Kubernetes 1.15: Alpha
```

Comment:
Can you put this in table format similar to the other feature pages?

Reply:
sorry, I missed that; added